src/bitcoin-cli.cpp

```diff
@@ -298,30 +296,11 @@ class GetinfoRequestHandler: public BaseRequestHandler
 class NetinfoRequestHandler : public BaseRequestHandler
 {
 private:
-    bool IsAddrIPv6(const std::string& addr) const
-    {
-        return !addr.empty() && addr.front() == '[';
-    }
-    bool IsInboundOnion(const std::string& addr_local, int mapped_as) const
-    {
-        return mapped_as == 0 && addr_local.find(ONION) != std::string::npos;
-    }
-    bool IsOutboundOnion(const std::string& addr, int mapped_as) const
-    {
-        const size_t addr_len{addr.size()};
-        const size_t onion_pos{addr.rfind(ONION)};
-        return mapped_as == 0 && onion_pos != std::string::npos && addr_len > ONION_LEN &&
-               (onion_pos == addr_len - ONION_LEN || onion_pos == addr.find_last_of(":") - ONION_LEN);
-    }
-    uint8_t m_details_level{0}; //!< Optional user-supplied arg to set dashboard details level
+    static const int m_networks = 3; // Number of possible networks
```

Comment:
5604a61435c5dfdeed41db5eca489b353dee0b99
```suggestion
    static constexpr int m_networks = 3; //!< Number of possible networks.
```

Reply:
Done